### DISCUSSION TITLE
Monorepo shared files

### DISCUSSION THREAD

**Discussion by coderinblack08** [vercel/vercel#5329] [Help] [answered]
I am trying to import a function from a shared package called `@pluto/common`. I have checked the `Include source files outside of the Root Directory` checkbox and have the root dir set to packages/client.

```ts
...
import { loginSchema } from '@pluto/common';
...
```

It is giving me an error:

<img width="917" alt="Screen Shot 2020-10-25 at 5 10 05 PM" src="https://user-images.githubusercontent.com/41022901/97122685-ee390180-16e4-11eb-9641-976ca9379ed1.png">

Thanks!

**Comment by yassinebridi** [comment, marked as answer]
You can commit the build files of `@pluto/common`(`dist` for example), and vercel will know where to get the `@pluto/common` declarations.

**Comment by coderinblack08** [comment]
Thanks so much! I thought I pushed the dist folder 👍
